projects/optic/src/commands/api/add.ts

```diff
@@ -121,6 +121,14 @@ async function promptForStandard(
 ): Promise<string | undefined> {
   const existingStandards = await client.getOrgStandards(orgId);
 
+  // if empty, create default without interrupting first time users
+  if (existingStandards.length === 0) {
+    const standard = await client.createOrgStandard(orgId, [
+      { name: 'breaking-changes', config: {} },
+    ]);
+    return standard.slug;
+  }
```

Comment:
Also - intended to be here? If yes then we might want to add some logging?

Reply:
No. I don't think we want people to have to think about standards on first run.